addons/medical_ai/StateMachine.hpp

```diff
@@ -1,5 +1,5 @@
 class GVAR(stateMachine) {
-    list = "allUnits select {local _x}";
+    list = "allUnits select {local _x && {_x != ACE_player} && {!isPlayer _x}}";
```

Comment:
if you are remote controlling a unit, your player unit is not `ACE_player` but your remote controlled AI is.